test/SPARQLProtocolTest.cpp

```diff
@@ -488,16 +487,57 @@ TEST(SPARQLProtocolTest, parseHttpRequest) {
       parse(makePostRequest("/", URLENCODED, "access-token=foo&query=bar")),
       ParsedRequestIs("/", "foo", {{"access-token", {"foo"}}},
                       Query{"bar", {}}));
+  // We add the leading `/` if it is missing.
+  EXPECT_THAT(parse(makeGetRequest("?query=foo")),
+              ParsedRequestIs("/", std::nullopt, {}, Query{"foo", {}}));
```

Comment:
Still the question: Is that valid HTTP that a client should be able to send?

Reply:
No, but until the conformance tests are fixed, we should imo still accept it.